test/unit/api_spec.js

```diff
@@ -2322,5 +2322,27 @@ describe("api", function () {
         await loadingTask.destroy();
       }
     );
+
+    it("write a value in an annotation, save the pdf and load it", async function () {
```

Comment:
Nit: Sorry, but this unit-test ended up in the "PDFDataRangeTransport"-block which isn't a great fit. Can you please move it to e.g. here instead: https://github.com/mozilla/pdf.js/blob/c914e9f0a631b92ed81d1a685b42ae088adefcdc/test/unit/api_spec.js#L1285